src/components/MemberInviteList.js

```diff
@@ -0,0 +1,191 @@
+import PropTypes from 'prop-types';
```

Comment:
@tienifr thoughts on renaming this to UserInviteList? Member wording is used when inviting a member to a workspace but not really when inviting users to a room. I think User is the correct word to use as this is a generic way of inviting users to something. thoughts?